src/projectM-sdl/projectM_SDL_main.cpp

```diff
@@ -1,104 +1,131 @@
-* modify it under the terms of the GNU Lesser General Public
-* License as published by the Free Software Foundation; either
-* version 2.1 of the License, or (at your option) any later version.
-*
-* This library is distributed in the hope that it will be useful,
-* but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-* Lesser General Public License for more details.
-*
-* You should have received a copy of the GNU Lesser General Public
-* License along with this library; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
-* See 'LICENSE.txt' included within this release
-*
-* projectM-sdl
-* This is an implementation of projectM using libSDL2
-* 
-* main.cpp
-* Authors: Created by Mischa Spiegelmock on 6/3/15.
-*
-* 
-*	RobertPancoast77@gmail.com :
-* experimental Stereoscopic SBS driver functionality
-* WASAPI looback implementation
-* 
-*
-*/
+ * projectM -- Milkdrop-esque visualisation SDK
+ * Copyright (C)2003-2019 projectM Team
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ * See 'LICENSE.txt' included within this release
+ *
+ * projectM-sdl
+ * This is an implementation of projectM using libSDL2
+ *
+ * main.cpp
+ * Authors: Created by Mischa Spiegelmock on 6/3/15.
+ *
+ *
+ *	RobertPancoast77@gmail.com :
+ * experimental Stereoscopic SBS driver functionality
+ * WASAPI looback implementation
+ *
+ *
+ */
 
 #include "pmSDL.hpp"
 
 #if OGL_DEBUG
-void DebugLog(GLenum source,
-               GLenum type,
-               GLuint id,
-               GLenum severity,
-               GLsizei length,
-               const GLchar* message,
-               const void* userParam) {
-
-    /*if (type != GL_DEBUG_TYPE_OTHER)*/ 
+void DebugLog(GLenum source, GLenum type, GLuint id, GLenum severity, GLsizei length,
+	const GLchar *message, const void *userParam)
+{
+
+	/*if (type != GL_DEBUG_TYPE_OTHER)*/
 	{
-        std::cerr << " -- \n" << "Type: " <<
-           type << "; Source: " <<
-           source <<"; ID: " << id << "; Severity: " <<
-           severity << "\n" << message << "\n";
-       }
- }
+		std::cerr << " -- \n"
+							<< "Type: " << type << "; Source: " << source << "; ID: " << id
```

Comment:
what happened here?

Reply:
It looks like Visual Studio tried to be "helpful" and changed the formatting. I'm not sure how to easily undo that unless I just submit a new PR and manually / copy paste the changes to the relevant sections.